shared/login/forms/index.native.js

```diff
@@ -8,61 +8,61 @@ import {
   Text,
   NativeScrollView,
   NativeDimensions,
+  HOCTimers,
+  type PropsWithTimer,
 } from '../../common-adapters/index.native'
 import {globalColors, globalStyles, globalMargins} from '../../styles'
 
 import type {Props} from '.'
-import HOCTimers, {type TimerProps} from '../../common-adapters/hoc-timers'
 
 const Scroller = (props: any) => (
   <NativeScrollView style={{height: '100%', width: '100%', paddingTop: 20}}>
     {props.children}
   </NativeScrollView>
 )
 
-const Splash = HOCTimers(
```

Comment:
lets avoid this style where everything is indented cause we're wrapper it here, just wrap it below